examples/convecting_plankton.jl

```diff
@@ -214,13 +214,13 @@ avg_P_timeseries = FieldTimeSeries(filepath, "avg_P")
 
 times = w_timeseries.times
 buoyancy_flux_time_series = [buoyancy_flux(0, 0, t, buoyancy_flux_parameters) for t in times]
-nothing # hide
+nothing #hide
```

Comment:
fyi I prefer `# hide` with a space. 

https://documenter.juliadocs.org/stable/man/syntax/#@example-block

Reply:
For Literate.jl I think we _need_ to avoid the space.
https://fredrikekre.github.io/Literate.jl/v2/fileformat/#Filtering-lines

Looking a the docs, the lines with `# hide` were not being hidden...

Reply:
oh ok